assets/src/components/vehiclePropertiesPanel.tsx

```diff
@@ -13,9 +14,22 @@ const VehiclePropertiesPanel = ({ selectedVehicle }: Props) => {
 
   const hideMe = () => dispatch(deselectVehicle())
 
+  useEffect(() => {
+    const handleSwipe = (swipeDirection: SwipeDirection) => {
+      if (swipeDirection === "Right") {
+        hideMe()
+      }
+    }
+
+    return detectSwipe("m-vehicle-properties-panel", handleSwipe)
+  })
+
   return (
     <>
-      <div className="m-vehicle-properties-panel">
+      <div
+        id="m-vehicle-properties-panel"
```

Comment:
If you swipe over the grayed overlay, does that close the panel? Would we want that? If the answers are no and yes, could we move the handler up a level so it covers both?

Reply:
No and no. You only need to tap in the grayed overlay to close the panel.